composer.json

```diff
@@ -24,7 +24,8 @@
   "autoload": {
     "psr-4": {
       "Terminus\\": ["php/", "php/Terminus/"],
-      "Pantheon\\Terminus\\": "src/"
+      "Pantheon\\Terminus\\": "src/",
+      "Pantheon\\Terminus\\Tests\\": "tests/"
```

Comment:
@greg-1-anderson I can't get PHPUnit to load it automatically:

```
dustinleblanc  ⋯  code  pantheon-systems  terminus  ./vendor/bin/phpunit -c tests/config/phpunit-10.xml.dist tests/new_unit_tests/Commands/ArtCommandTest.php

PHP Fatal error:  Class 'Pantheon\Terminus\UnitTests\Commands\CommandTestCase' not found in /Users/dustinleblanc/code/pantheon-systems/terminus/tests/new_unit_tests/Commands/ArtCommandTest.php on line 5

Fatal error: Class 'Pantheon\Terminus\UnitTests\Commands\CommandTestCase' not found in /Users/dustinleblanc/code/pantheon-systems/terminus/tests/new_unit_tests/Commands/ArtCommandTest.php on line 5
```
Not entirely sure the best way to go, It looks like @TeslaDethray mentioned some toher stuff going on around this yesterday too